PEERS_GUIDELINES.md

```diff
@@ -0,0 +1,189 @@
+  - [@clovislima](https://github.com/clovislima)
+- French (`fr`) content — the
+[@yari-content-fr](https://github.com/orgs/mdn/teams/yari-content-fr)
+  team, which consists of:
+  - [@nicolas-goudry](https://github.com/nicolas-goudry)
+  - [@JNa0](https://github.com/JNa0)
+  - [@tristantheb](https://github.com/tristantheb)
+  - [@LEMIBANDDEXARI](https://github.com/LEMIBANDDEXARI)
+  - [@SphinxKnight](https://github.com/SphinxKnight)
+- Japanese (`ja`) content — the
+[@yari-content-ja](https://github.com/orgs/mdn/teams/yari-content-ja)
+  team, which consists of:
+  - [@hmatrjp](https://github.com/hmatrjp)
+  - [@potappo](https://github.com/potappo)
+  - [@dynamis](https://github.com/dynamis)
+  - [@kenji-yamasaki](https://github.com/kenji-yamasaki)
+  - [@mfuji09](https://github.com/mfuji09)
+- Korea (`ko`) content — the
+[@yari-content-ko](https://github.com/orgs/mdn/teams/yari-content-ko)
+  team, which consists of:
+  - [@yechoi42](https://github.com/yechoi42)
+  - [@cos18](https://github.com/cos18)
+  - [@GwangYeol-Im](https://github.com/GwangYeol-Im)
+  - [@pje1740](https://github.com/pje1740)
+  - [@nKiNk](https://github.com/nKiNk)
+  - [@yujo11](https://github.com/yujo11)
+  - [@hochan222](https://github.com/hochan222)
+- Russian (`ru`) content — the
+[@yari-content-ru](https://github.com/orgs/mdn/teams/yari-content-ru)
+  team, which consists of:
+  - [@armanpwnz](https://github.com/armanpwnz)
+  - [@captainspring](https://github.com/captainspring)
+  - [@mpstv](https://github.com/mpstv)
+  - [@myshov](https://github.com/myshov)
+  - [@Saionaro](https://github.com/Saionaro)
+  - [@sashasushko](https://github.com/sashasushko)
+  - [@lex111](https://github.com/lex111)
+- Chinese (`zh-CN` and `zh-TW`) content — the
+[@yari-content-zh](https://github.com/orgs/mdn/teams/yari-content-zh)
+  team, which consists of:
+  - [@t7yang](https://github.com/t7yang)
+  - [@dibery](https://github.com/dibery)
+  - [@irvin](https://github.com/irvin)
+
+---
+
+## Synchronization with the en-US document structure
+
+Before unfreezing the Tier 1 locales, we made an update to synchronize all the
+localized document tree structures with the `en-US` tree structure (English
+slugs only), to make the documentation easier to manage. This resulted in two
+new buckets of documents being created for each locale, existing as
+subdirectories of each local folder:
+
+- `orphaned` — documents that are not associated with any parent `en-US` page.
+- `conflicting` — documents with duplicate translations in existence (e.g.
+  localized once under the existing `en-US` slug, and then again under a
+  localized slug). The duplicate(s) are put in this folder.
+
+Active locale maintenance teams are invited to spend some time exploring the
+orphaned and conflicting documents, to see whether any of this work is worth
+keeping (either adding to, or merging with an existing document in, the main
+tree), or whether it can just be deleted.
+
```

Comment:
This whole section is now speaking about a historic past that no one new will have the capacity to care about or understand. 

Can we rewrite it so it simply explains what it means? Perhaps a simple intro followed by the bullet points there. It can be mentioned that `conflicting` shouldn't happen anymore but was a relic from early migration. `orphaned` documents will continue to come into existence when en-US documents are removed which happens periodically.